PackTravel/urls.py

```diff
@@ -38,5 +38,6 @@
     path('logout/', LogoutView.as_view()),
     path('myrides/', userView.my_rides, name = 'search'),
     path('delete_ride/<ride_id>', userView.delete_ride, name = 'delete_ride'),
-    path('u/<userid>', userView.user_profile, name = 'user_profile')
+    path('u/<userid>', userView.user_profile, name = 'user_profile'),
+    path('edit/', userView.edit_user, name = 'user_user')
```

Comment:
Update url to edit-user or edit-profile. Something like that. Edit is very vague